src/io/jesi/backpack/string.cljc

```diff
@@ -138,9 +138,12 @@
 (defn split-at-first
   "Splits s at the first occurrence of value"
   [value s]
-  (when (and (not-blank? value) (not-blank? s))
-    (when-let [idx (str/index-of s value)]
-      (let [val-length (count value)
-            before-split (subs s 0 idx)
-            after-split (subs s (+ idx val-length))]
-        [before-split after-split]))))
+  (cond
+    (str/blank? s) []
+    (str/blank? value) [s]
```

Comment:
There is an issue with using `blank?` What if we pass an empty string along?

`(str/blank? "   ")` => true

What if we want to split on the first `\space`?

`(and (string? value) (nil? (seq value)))` Might be what you want

Reply:
Ahh, makes sense... this is why you're earning the big bucks

Although I do like the idea of supporting chars since we're encouraged to use them throughout our code habitually (even though I pretty much undo)

Reply:
You can support chars by converting the value to a string before use.

`(let [value (str value)] ...`

Reply:
Which You've done. So I'm good with that.